src/components/shared/Button/index.module.scss

```diff
@@ -31,7 +31,16 @@ button {
 .smallLinkText {
     font-family: 'Open Sans', sans-serif;
     font-size: 0.688rem;
-    margin-top: 0.938rem;
     text-transform: none;
     letter-spacing: normal;
 }
+
+.mediumLinkText {
+    @extend .smallLinkText;
```

Comment:
you could create a class called` linkText` and extend that one to small, medium, etc

Reply:
Done!